### PR TITLE
Add details

### PR DESCRIPTION
**Pull request by ebonynon**
Update board details

### CODE REVIEW

File: src/assets/data/board.json [hunk 1]
```diff
@@ -15,10 +15,20 @@
...
-      "twitter": "https://twitter.com/iamSafnaj",
-      "instagram": "https://www.instagram.com/iam_safnaj/",
-      "linkedin": "https://www.linkedin.com/in/ahamed-safnaj"
+      "facebook": "http://facebook.com/chamodshehanka",
+      "twitter": "http://twitter.com/chamodshehanka",
+      "instagram": "http://instagram.com/chamodshehanka/",
+      "linkedin": "http://linkedin.com/in/chamodshehanka/"
+    },
+    {
+      "id": 3,
```
Comment (Safnaj) [inline src/assets/data/board.json:24]: Make ur id as 10

**Review by Safnaj — CHANGES_REQUESTED** [review]

**Comment by Shehanka** [comment]
Fetch images from GitHub profile. Like we did on GC Sri Lanka @Safnaj

**Review by Shehanka — APPROVED** [review]

**Comment by Safnaj** [comment]
> Fetch images from GitHub profile. Like we did on GC Sri Lanka @Safnaj 

If we changed GitHub profile in future it will be a issue. Better to have separate images for this.

**Comment by Shehanka** [comment]
> > Fetch images from GitHub profile. Like we did on GC Sri Lanka @Safnaj
> 
> If we changed GitHub profile in future it will be a issue. Better to have separate images for this.

noo if we change it automatically picture will update

**Comment by Safnaj** [comment]
> > > Fetch images from GitHub profile. Like we did on GC Sri Lanka @Safnaj
> > 
> > 
> > If we changed GitHub profile in future it will be a issue. Better to have separate images for this.
> 
> noo if we change it automatically picture will update

are you sure ?? i faced same issue when using github profile picture URL. Also we had same problem in GitHub Community Site

**Review by Shehanka — APPROVED** [review]

**Pull request merged**